pkg/proc/gdbserial/rr.go

```diff
@@ -238,6 +239,11 @@ func rrStderrParser(stderr io.ReadCloser, initch chan<- rrInit, quiet bool) {
 			close(initch)
 			break
 		}
+		if strings.HasPrefix(line, rrGdbCommandLegacyPrefix) {
```

Comment:
Can you fold this into the above branch since the branch contents are essentially the same aside from the prefix used to reslice.

Reply:
For example:

```go
	prompt := "  'gdb' '-l' '10000' '-ex' 'set sysroot /' '-ex' 'target extended-remote 127.0.0.1:12007'"
	prefix1 := "  gdb "
	prefix2 := "  'gdb' "
	flags := ""
	foundPrefix := false
	if flags, foundPrefix = strings.CutPrefix(prompt, prefix1); !foundPrefix {
		flags, foundPrefix = strings.CutPrefix(prompt, prefix2)
	}
	if foundPrefix {
		initch <- rrParseGdbCommand(flags)
		close(initch)
		break
	}
```

Reply:
Thanks! Done and pushed up a commit